### ISSUE TITLE
Error `VersionError The requested version (30) is less than the existing version (40)`

### ISSUE THREAD

**Issue by yorch** [boydaihungst/altair-graphql-plugin-github-sync#1]
Thanks for this plugin! I'm trying to use it for the first time, using the Altair Web app, installed the plugin, filled the fields for the plugin (API key, Gist name) and when clicking on the Upload button, I'm getting the error:

```
VersionError The requested version (30) is less than the existing version (40)
```

<img width="968" alt="image" src="https://user-images.githubusercontent.com/60499/160506947-3c01e425-d4fa-41d1-9b17-686a983cbdc8.png">

Seems to be related to the Altair IndexedDB database version, but not really sure where the mismatch between versions come from. I dug into the plugin code and didn't find anything obvious. From my quick debugging, I know the code runs at least till this point: https://github.com/boydaihungst/altair-graphql-plugin-github-sync/blob/master/src/providers/GitHubSyncProvider.ts#L43

<img width="481" alt="image" src="https://user-images.githubusercontent.com/60499/160507004-f53c3acf-f074-4f56-814e-ad382852887f.png">

**Comment by boydaihungst** [comment]
Thanks for reporting. I've fixed the issue in version 1.0.5. Try to reinstall plugin.
![image](https://user-images.githubusercontent.com/38396158/160533780-6dedd48c-14b5-410d-a6be-c29d1c1b400d.png)

**Comment by yorch** [comment]
Thanks! working now.. just for reference, the best way to get the latest version of the plugin is to clear the cache (or force a clean reload) so the browser can re-download the plugin assets (as it's not a versioned URL: https://cdn.jsdelivr.net/npm/altair-graphql-plugin-github-sync@latest/dist/github-sync.umd.js), no need to re-install
